### ISSUE TITLE
Emoji picker can only be closed by picking an emoji

### ISSUE THREAD

**Issue by with-heart** [Soil-labs/eden-app-2-FrontEnd#704]
It's entirely possible that users will open this by accident and should be able to close it by clicking away from it or hitting <kbd>esc</kbd>.
